sample-lnd.conf

```diff
@@ -1282,8 +1282,8 @@
 ; Set to enable support for the experimental taproot channel type.
 ; protocol.simple-taproot-chans=false
 
-; Set to storing backup data for other peers.
-; protocol.peer-storage=true
+; Set to enable storing backup data for other peers.
+; protocol.peer-storage=false
```

Comment:
This should be consolidated with the previous commit and @ellemouton is right that this should be put into the commit that introduces the option.